hacktoberfest_challenges/model_license_other.md

```diff
@@ -74,7 +74,7 @@ Here is the list of models with `license:other` and a LICENSE file in the repo.
 |------|----------------------------------------------------------------------------|--------|------------|-------------------------|----------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|
 |      |                                                                            |[decapoda-research/llama-7b-hf](https://huggingface.co/decapoda-research/llama-7b-hf)|379224      |1223                     |llama-license                                                                                 |https://huggingface.co/decapoda-research/llama-7b-hf/blob/main/LICENSE                        |[LICENSE](https://huggingface.co/decapoda-research/llama-7b-hf/blob/main/LICENSE)                  |                                                                                                                     |                                                                                   |
-|      |                                                                            |[huggyllama/llama-7b](https://huggingface.co/huggyllama/llama-7b)|319220      |200                      |llama-license                                                                                 |https://huggingface.co/huggyllama/llama-7b/blob/main/LICENSE                                  |[LICENSE](https://huggingface.co/huggyllama/llama-7b/blob/main/LICENSE)                            |                                                                                                                     |                                                                                   |
+|Opened|[HUB_PR](https://huggingface.co/huggyllama/llama-7b/commit/de898aefce429a3f8e5000635357b79b5d9922b0)|[huggyllama/llama-7b](https://huggingface.co/huggyllama/llama-7b)|319220      |200                      |llama-license                                                                                 |https://huggingface.co/huggyllama/llama-7b/blob/main/LICENSE                                  |[LICENSE](https://huggingface.co/huggyllama/llama-7b/blob/main/LICENSE)                            |                                                                                                                     |                                                                                   |
```

Comment:
PR on the Hub looks good to me! Thanks for opening it ❤️

As for this PR, could you put the PR url instead of the commit url? (i.e. https://huggingface.co/huggyllama/llama-7b/discussions/7). Thanks in advance!

Reply:
Thanks for review! Sure, I will update PR url.

Reply:
@Wauplin Please review, I have done the changes.